flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "publish-delay-hours": 1
```

Comment:
Any reason to change it from the default?

Reply:
Just a personal preference.